tests/test_agent_name.py

```diff
@@ -24,8 +24,8 @@ def test_add_inline_agent_name():
     assert result.name == "assistant"
 
 
-def test_add_inline_agent_name_content_blocks():
-    content_blocks = [
+def test_add_inline_agent_name_content_blocks() -> None:
+    content_blocks: list[str | dict] = [
```

Comment:
this should be just list of dict

Reply:
`content_blocks` is passed as the `content` parameter to `AIMessage` below, and [that expects `list[str | dict]`](https://github.com/langchain-ai/langchain/blob/fce8caca16121024547fb0e8eb2d289c8f96396a/libs/core/langchain_core/messages/ai.py#L179). Because `list` is invariant, mypy won't accept a `list[str]` passed to a `list[str | dict]`.

Perhaps it would be better for `AIMessage` to expect `list[str] | list[dict]` (or even `Sequence[str | dict]`, since `Sequence` is covariant) to avoid needing to do this, but that would require a change in that library. Happy to do that if you think that's preferable.

Reply:
ah, good point re: `AIMessage`, let's keep as is then!